src/crispy/boxed.h

```diff
@@ -38,7 +38,7 @@ namespace helper
 // }}}
 
 template <typename T>
-constexpr bool is_boxed = helper::is_boxed<T>::value;
+constexpr bool IsBoxed = helper::is_boxed<T>::value;
```

Comment:
boxed.h should retain it's C++/STL like naming conventions, maybe?

Reply:
this file will be deleted soon, no need to update PR